contracts/scripts/operational/setRemoteTokenBridgeTask.ts

```diff
@@ -38,8 +32,8 @@ task(
       "remoteTokenBridgeAddress",
       "REMOTE_TOKEN_BRIDGE_ADDRESS",
     );
+
     let tokenBridgeAddress = getTaskCliOrEnvValue(taskArgs, "tokenBridgeAddress", "TOKEN_BRIDGE_ADDRESS");
-    let tokenBridgeProxyAdmin = taskArgs.tokenBridgeProxyAdminAddress;
 
     if (tokenBridgeAddress === undefined) {
```

Comment:
Could you use `!tokenBridgeAddress` instead of `tokenBridgeAddress === undefined` please ?

Reply:
done